src/main/java/org/quantumbadger/redreader/http/HTTPBackend.java

```diff
@@ -92,16 +92,10 @@ public static String encodeList(final List<PostField> fields) {
 	}
 
 	interface Listener {
+		void onError(@CacheRequest.RequestFailureType int failureType, Throwable exception, Integer httpStatus);
 
-		void onError(RequestFailureType failureType, Throwable exception, Integer httpStatus);
-
-		void onSuccess(
-				String mimetype,
-				Long bodyBytes,
-				InputStream body);
+		void onSuccess(String mimetype, Long bodyBytes, InputStream body);
```

Comment:
No need to change it back now, just something to bear in mind in future :)

Ideally people can rebase their changes on the latest master with no manual work -- if the formatting of the code has changed in the meantime this won't be the case.